dataland-frontend/src/components/resources/companiesOnlySearch/CompaniesOnlySearchBar.vue

```diff
@@ -83,20 +80,22 @@ export default defineComponent({
      * @param event object containing the stored company
      * @param event.value the stored company object
      */
-    pushToChooseFrameworkForDataUploadPageForItem(event: { value: StoredCompany }) {
-      void this.$router.push(`/companies/${event.value.companyId}/frameworks/upload`);
+    async pushToChooseFrameworkForDataUploadPageForItem(event: AutoCompleteItemSelectEvent) {
+      await this.$router.push(
+        `/companies/${assertDefined(assertDefined(event.value as CompanyIdAndName).companyId)}/frameworks/upload`
+      );
     },
     /**
      * Queries the getCompanies endpoint and writes the response to the variable autoCompleteArray
      * @param companyName object containing the search query for the getCompanies endpoint
      * @param companyName.query the query for the getCompany endpoint
      */
-    async searchCompanyName(companyName: { query: string }) {
+    async searchCompanyName(companyName: AutoCompleteCompleteEvent) {
```

Comment:
Wenn der input als komplettes event hier reingenommen wird, dann sollte `companyName` entsprechend umbenannt werden.

Reply:
Richting. Done. To be pushed.

Reply:
👍